google_compute_engine/distro/utils.py

```diff
@@ -37,3 +39,13 @@ def EnableNetworkInterfaces(
       dhclient_script: string, the path to a dhclient script used by dhclient.
     """
     pass
+
+  def HandleClockSync(self, logger):
+    """Sync clock.
+
+    Called when clock drift token changes.
+
+    Args:
+      logger: logger object, used to write to SysLog and serial port.
+    """
+    helpers.CallHwclock(logger)
```

Comment:
/cc @ryanwe 

This file was intended as an interface. The function definitions belong here, but the implementation exists in the distro specific directories. Using this paradigm, introducing a new function definition requires an update to each distro utils file. Not sure if we should change the design, but we should give it some thought.